tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -15,15 +15,30 @@ parameters:
   default: true
 
 - name: DoEsrp
-  displayName: Run code sign tasks? Must be true if you are doing an Onnx Runtime release.
+  displayName: Run code sign tasks? Must be true if you are doing an ONNX Runtime release
   type: boolean
   default: true
 
 - name: IsReleaseBuild
-  displayName: Is a release build? Set it to true if you are doing an Onnx Runtime release.
+  displayName: Is a release build? Set it to true if you are doing an ONNX Runtime release.
   type: boolean
   default: false
 
+- name: PreReleaseVersionSuffixString
+  displayName: Suffix added to pre-release package version. Only used if IsReleaseBuild is true. Denotes the type of pre-release package.
+  type: string
+  values:
+  - -alpha
+  - -beta
+  - -rc
+  - none
+  default: none
+
+- name: PreReleaseVersionSuffixNumber
+  displayName: Number added to pre-release package version. Only used if IsReleaseBuild is true. Denotes the sequence of a pre-release package.
+  type: number
+  default: 0
+
```

Comment:
This seems potentially error prone. How does someone running this discover what is/isn't allowed in the suffix and what the last value/s used where?

Is there a reason not to use a timestamp and commit id as the suffix like we do with the nightly packages? That's guaranteed to result in the latest package being used, provides important information on what's in the package, and will be guaranteed to sort correctly when nuget looks to see if there are updated packages.

Reply:
] Azure Devops's web ui will show the "PreReleaseVersionSuffixString" parameter as a drop down box. So it's easy to use. It won't let you type strings there.

Reply:
Not having the date time string was the requested developer experience by the user. Open to suggestions to make it less error prone. The generation is manual, as is the nuget upload (for better or worse), so whoever is running the pipeline should be aware of the required number.